static/app/views/organizationStats/usageChart/index.tsx

```diff
@@ -522,6 +560,14 @@ function UsageChartBody({
         top: 5,
         data: chartLegendData(),
         theme,
+        selected: {
+          [SeriesTypes.ACCEPTED]: true,
+          [SeriesTypes.FILTERED]: true,
+          [SeriesTypes.RATE_LIMITED]: true,
+          [SeriesTypes.INVALID]: true,
+          [SeriesTypes.CLIENT_DISCARD]: false,
```

Comment:
`client_discard` is undoubtedly useful, but it occurs frequently. If not disabled by default, it could clutter the chart. It's important to remember that while the stats page can be used for debugging, that's not its primary purpose.

Reply:
Looking at this now, it's not obvious for customers that this can be enabled. Could we find a more prominent placement for this? I'm still in favor of hiding those by default (at least when "Transactions" are selected), but maybe something like this would work? 

<img width="763" alt="image" src="https://github.com/getsentry/sentry/assets/1433023/79e5fa93-293c-42f8-84b6-d39c4db75668">

Reply:
this will be addressed in a separate PR

Reply:
here is the PR https://github.com/getsentry/sentry/pull/74248